docs/theme-schema.json

```diff
@@ -0,0 +1,156 @@
+                    "additionalProperties": false
+                },
+                "textcolor": {
+                    "type": "object",
+                    "properties": {
+                        "h": {
+                            "type": "integer",
+                            "maximum": 360,
+                            "minimum": 0
+                        },
+                        "s": {
+                            "type": "integer",
+                            "maximum": 100,
+                            "minimum": 0
+                        },
+                        "l": {
+                            "type": "integer",
+                            "maximum": 100,
+                            "minimum": 0
+                        }
+                    },
+                    "required": ["h", "s", "l"],
+                    "additionalProperties": false
+                },
+                "toolbar_field": {
+                    "type": "object",
+                    "properties": {
+                        "h": {
+                            "type": "integer",
+                            "maximum": 360,
+                            "minimum": 0
+                        },
+                        "s": {
+                            "type": "integer",
+                            "maximum": 100,
+                            "minimum": 0
+                        },
+                        "l": {
+                            "type": "integer",
+                            "maximum": 100,
+                            "minimum": 0
+                        },
+                        "a": {
+                            "type": "number",
+                            "maximum": 100,
+                            "minimum": 0
+                        }
+                    },
+                    "required": ["h", "s", "l", "a"],
+                    "additionalProperties": false
+                },
+                "toolbar_field_text": {
+                    "type": "object",
+                    "properties": {
+                        "h": {
+                            "type": "integer",
+                            "maximum": 360,
+                            "minimum": 0
+                        },
+                        "s": {
+                            "type": "integer",
+                            "maximum": 100,
+                            "minimum": 0
+                        },
+                        "l": {
+                            "type": "integer",
+                            "maximum": 100,
+                            "minimum": 0
+                        }
+                    },
+                    "required": ["h", "s", "l"],
+                    "additionalProperties": false
+                }
+            },
+            "required": ["toolbar", "toolbar_text", "accentcolor", "textcolor", "toolbar_field", "toolbar_field_text"]
+        }
+    },
+    "required": ["title", "colors"]
```

Comment:
Same as above, we should add `"additionalProperties": false` here too (which should cause a few more linter errors, if my guess is correct).

Reply:
In that case both `default.json` and `005.json` is invalidated.

Reply:
Yay, my guess is correct!
We'll need to update the schema to allow for those other optional properties (`images.additional_backgrounds[]` and `images.headerURL`, IIRC).